.travis.yml

```
@@ -2,7 +2,7 @@ language: php
 
 sudo: false
 
-php: [5.6, 7.0, 7.1, 7.2, 7.3]
+php: [5.6, 7.0, 7.1, 7.2, 7.3, 7.4]
```

Could you also add tests with PHP 8.0?
```suggestion
php: [5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0snapshot]
```